src/ElasticSearchIndexer.php

```diff
@@ -214,4 +223,15 @@ private function extendBatch(string $action, string $indexName, IndexDocument $d
 
         return $params;
     }
+
+    private function parseErrors(array $clientResponse): string
+    {
+        $errors = '';
+        
+        foreach ($clientResponse['items'] as $response) {
+            $response = reset($response);
```

Comment:
why reset the response?

Reply:
For each action that we pass to a bulk method, we will receive an array with one element  that will contain an error message.